langchain4j-agentic/src/main/java/dev/langchain4j/agentic/supervisor/SupervisorAgentServiceImpl.java

```diff
@@ -37,9 +37,15 @@ public T build() {
             });
         }
 
-        return build(() -> new SupervisorPlanner(chatModel, chatMemoryProvider, maxAgentsInvocations,
-                contextStrategy, responseStrategy, requestGenerator,
-                outputKey, output));
+        return build(() -> new SupervisorPlanner(
```

Comment:
It would be great if you could revert this. Please don't change the formatting in classes were you didn't change anything else, otherwise it will be harder to review this pull request.